frame/assets/Cargo.toml

```diff
@@ -37,6 +37,7 @@ std = [
 	"codec/std",
 	"sp-std/std",
 	"sp-runtime/std",
+	"pallet-balances/std",
```

Comment:
Actually this should be removed and just remove `default-features = false` from the dev dependency to get `std`

Reply:
ah yes, fixed in https://github.com/paritytech/substrate/pull/7950/commits/68307c9ab4477503053a22a26e9015681ba92fad